src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -180,7 +180,7 @@
 
     @Argument(
             fullName = "INTERVAL_ITERATOR",
-            doc = "Iterate through the files assuming they are a pre-created subset interval of the full genome.  " +
+            doc = "Iterate through the file assuming they are a pre-created subset interval of the full genome.  " +
```

Comment:
...." assuming it consists of a pre-created subset ...."

Reply:
Fixe.d